lib/restful_model_collection.rb

```diff
@@ -138,8 +138,11 @@ def get_model(id)
 
     def get_model_collection(offset = 0, limit = 100)
       filters = @filters.clone
-      filters[:offset] = offset
-      filters[:limit] = limit
+
+      # If filters have already been set for limit or offset, ignore the
```

Comment:
This is pretty confusing. I'd prefer that we handle `limit` and `offset` a level higher (i.e: inside `range`, `each`, etc.)